.github/workflows/api-update.yaml

```diff
@@ -1,11 +1,11 @@
-name: Update SDK
+name: Update API specs 
 on:
   schedule:
     - cron: '0 0 * * *'
   ## Manual request that can be used for testing and retriggering failed updates
   workflow_dispatch:
```

Comment:
we should also have this event https://github.com/redhat-developer/app-services-sdk-go/blob/0eee56db7292d762196cd7bc317d8e40ba891e47/.github/workflows/update_openapi.yaml#L3-L4